router/http/src/main/scala/io/buoyant/router/http/Identifier.scala

```diff
@@ -0,0 +1,16 @@
+package io.buoyant.router.http
+
+import com.twitter.finagle.buoyant.Dst
+import com.twitter.finagle.http.Request
+import com.twitter.finagle.{Dtab, Path}
+import com.twitter.util.{Future, Try}
+
+object Identifier {
+  def dstConcrete(req: Request, baseDtab: () => Dtab): Option[Future[(Dst, Request)]] = {
+    req.headerMap.get("l5d-dst-concrete").flatMap { dst =>
```

Comment:
but this isn't actually concrete, is it? It's going to be applied through a dtab.